ImageFilterEditor/ImageFilterEditor/ImagePostViewController.swift

```diff
@@ -0,0 +1,47 @@
+//
+//  ViewController.swift
+//  ImageFilterEditor
+//
+//  Created by Shawn Gee on 5/4/20.
+//  Copyright © 2020 Swift Student. All rights reserved.
+//
+
+import UIKit
+
+class ImagePostViewController: UIViewController {
+    
+    // MARK: - IBOutlets
+    
+    @IBOutlet private weak var imageView: UIImageView!
+    @IBOutlet weak var imageFilterContainerHeightConstraint: NSLayoutConstraint!
+    
+    // MARK: - Private Properties
+    
+    private var imageFilterController: ImageFilterController!
+    
+    // MARK: - View Lifecycle
+    
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        imageFilterController.inputImage = imageView.image
+    }
+    
```

Comment:
I haven't dealt with this method at all, so I'm not quite sure what's going on here, but I am very intrigued...!